src/status_im/contexts/wallet/send/routes/view.cljs

```diff
@@ -133,6 +137,14 @@
                                                            (.toFixed (/ input-amount
                                                                         conversion-rate)
                                                                      crypto-decimals)))
+             amount-in-crypto-bn                        (money/bignumber amount-in-crypto)
+             send-amount-in-crypto-bn                   (money/bignumber send-amount-in-crypto)
+             lock-higher-than-send-amount?              (if (and (money/bignumber? amount-in-crypto-bn)
+                                                                 (money/bignumber?
+                                                                  send-amount-in-crypto-bn))
```

Comment:
I find `-bn` to be a bit more confusing. Can we name it something else? perhaps we can just pass it as that? 🤔